.github/workflows/primer_run_main.yaml

```diff
@@ -9,7 +9,7 @@ on:
   push:
     branches:
       - main
-
+      - "maintenance/**"
```

Comment:
The primer will probably still get the data from `main`. I don't think this is necessary.

Reply:
```suggestion
```